pkg/build/builder/sti.go

```diff
@@ -259,7 +264,11 @@ func (s *S2IBuilder) Build() error {
 		return errors.New(buffer.String())
 	}
 
-	glog.V(4).Infof("Creating a new S2I builder with build config: %#v\n", describe.Config(config))
+	client, err := docker.NewEngineAPIClient(config.DockerConfig)
```

Comment:
Hmm, the docker daemon load issue is dealt with in the bowels of s2i; this PR is really just the bump.  The API changed to require passing in a docker client, hence this diff.  I'm not sure that really merits a comment (he says, having just written this explanation here).